canvas_modules/common-canvas/__tests__/toolbar/toolbar-test.js

```diff
@@ -107,15 +107,60 @@ describe("Toolbar renders correctly", () => {
 			.simulate("click");
 		expect(toolbarActionHandler.calledOnce).to.equal(false);
 	});
+
+	it("should render a Toolbar with medium size buttons", () => {
```

Comment:
Can you add tests to make sure `hasIconOnly` get applied to the buttons correctly for those without a label?